library/core/src/convert/mod.rs

```diff
@@ -466,9 +466,9 @@ pub trait Into<T>: Sized {
 /// Prefer using [`Into`] over using `From` when specifying trait bounds on a generic function.
 /// This way, types that directly implement [`Into`] can be used as arguments as well.
 ///
-/// The `From` is also very useful when performing error handling. When constructing a function
+/// The `From` trait is also very useful when performing error handling. When constructing a function
 /// that is capable of failing, the return type will generally be of the form `Result<T, E>`.
-/// The `From` trait simplifies error handling by allowing a function to return a single error type
+/// It simplifies error handling by allowing a function to return a single error type
 /// that encapsulate multiple error types. See the "Examples" section and [the book][book] for more
```

Comment:
Many subjects and objects were mentioned in the immediately previous sentence, so it's no longer clear what "It" would refer to. However, just saying "`From`" is acceptable. Also, I believe "encapsulate" here needs to be "encapsulates" due to... verb agreement, I believe? "that encapsulates _multiple error types_."
```suggestion
/// `From` simplifies error handling by allowing a function to return a single error type
/// that encapsulates multiple error types. See the "Examples" section and [the book][book] for more
```

Reply:
This makes sense. I did not catch the 2nd typo. lgtm :+1: